simul/platform/aws/awsManager.go

```diff
@@ -0,0 +1,97 @@
+package aws
+
+import (
+	"fmt"
+)
+
+//Instance represents EC2 Amazon instance
+type Instance struct {
+	// EC2 ID
+	ID *string
```

Comment:
you don't need a pointer for that kind of fields, a simple string field is just more Go-like

Reply:
Yeh, for some reason the go API uses pointers for almost anything. I made the ID pointer just for convenience, so I don't have to *,& all the time.

Reply:
Oo that's strange - all right, fair enough !